scripts/zipper.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess as sp
+
+import scripts.setup as setup
+from scripts.config import image_name
+
+def run(slides, local):
```

Comment:
Now that we use this logic multiple times it might be time for a cleanup of `scripts`.